src/renderer/components/EnhancedTable/tableUtil.js

```diff
@@ -44,5 +44,29 @@ export const sortData = (obj, sortBy, order, path) => {
 
   const sorted = orderBy(sortByValueArr, Object.keys(obj), [order]);
 
-  return sorted.map((a) => Object.keys(a));
+  return sorted.map(Object.keys);
 };
+
+const compareNamespaces = (first, second) => {
+  const nameA = first.name.toUpperCase();
+  const nameB = second.name.toUpperCase();
+  // sort namespaces alphabetically based on name,
+  // with the exception of always putting the "default" namespace at the top of the list.
+  if (nameA.includes('DEFAULT')) {
+    return -1;
+  }
+  if (nameB.includes('DEFAULT')) {
+    return 1;
+  }
+  if (nameA < nameB) {
+    return -1;
+  }
+  if (nameA > nameB) {
+    return 1;
+  }
+
+  return 0;
+};
```

Comment:
`String.includes()` does not do an exact match. We should look for the "default" namespace _exactly_, otherwise, the sort will get messed-up if a namespace includes the word "default". There can only be one "default" namespace.

Also, using `String.localeCompare()` might be safer than a straight numerical comparison.

So this could be as simple as:

```suggestion
const compareNamespaces = (first, second) => {
  if (first.name === 'default') {
    return -1;
  }

  if (second.name === 'default') {
    return 1;
  }

  return first.name.localeCompare(second.name);
};
```

Reply:
I wasn't sure about 'default' thanks. Good point, about `localeCompare` appreciated